style.css

```diff
@@ -0,0 +1,358 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Montserrat&family=Open+Sans:wght@400;500;600;700&family=Oswald&family=Poppins:wght@400;500;600;700&display=swap");
+
+* {
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  background-color: #e4e5e7;
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  overflow-x: hidden;
+  min-width: 375px;
+}
+
+.homepage {
+  background-image: url(Assets/files/header-background-mobile.jpg);
+  min-width: 60%;
+  height: 700px;
+  border-bottom-left-radius: 10%;
+  margin-top: -1rem;
+  margin-left: -0.5rem;
+  margin-right: 0;
+  overflow-x: hidden;
+}
+
+#header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 2rem;
+}
+
+.mylogo {
+  color: #6070ff;
+  width: 100%;
+  margin-left: 1.5rem;
+  align-items: center;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  overflow-x: hidden;
+}
+
+.hamburger {
+  cursor: none;
+  margin-right: 10%;
+}
+
+.home-body {
+  background-image: url(Assets/files/header\ shapes.png);
+  background-repeat: no-repeat;
+  min-width: 60%;
+  height: 604px;
+}
+
+.heading {
+  min-width: 300px;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 50px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  margin-left: 1.3rem;
+  padding-top: 6.5rem;
+}
+
+.paragraph {
+  min-width: 50%;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 1.3rem;
+  margin-top: -1.7rem;
+}
+
+.connect {
+  color: #7f8cff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 1.3rem;
+}
+
+#social-links {
+  display: flex;
+  padding-right: 11.7rem;
+  padding-top: 2%;
+  margin-left: -1.2rem;
+  justify-content: space-between;
+  list-style: none;
+  margin-top: 0.5%;
+}
+
+#works {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  column-gap: 5px;
+  list-style: none;
+}
```

Comment:
- [ ] [OPTIONAL] According to the project requirement, this  section should use the grid to layout its content 🔢 However, since the previous reviewer did not mention it, I will assign it as an OPTIONAL change. Though I **HIGHLY** recommend you to consider my recommendation. It will improve the structure of your code ❤️